File: botnanajs/examples/empty.js
"use strict";

var botnana = require("../index");

botnana.once("ready", function() {
  // Real-time script API
  var script = "empty  marker empty";
  botnana.motion.evaluate(script);
});

botnana.start("ws://192.168.7.2:3012");

File: botnanars/examples/empty.rs
extern crate botnanars;
use botnanars::Botnana;
use std::sync::{Arc, Mutex};

fn main() {
    let botnana = Arc::new(Mutex::new(Botnana::new().unwrap()));

    let btn = botnana.clone();
    botnana.lock().unwrap().once("ready", move |_| {
        let script = "empty marker empty";
        btn.lock().unwrap().evaluate(script);
    });

    botnana.lock().unwrap().start("ws://localhost:3012");
    loop {}
}
